### PR TITLE
[Documentation] Adding a CONTRIBUTING.md file

### PR DESCRIPTION
**Pull request by hallelujah**
Part of https://issues.jboss.org/browse/THREESCALE-285

Simple contributing.md

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+Assuming that you don't see any red, you're ready to go.
+
+## Submitting pull requests
+
+1. Create a new branch, please don't work in your `master` branch directly.
+1. Add failing tests for the change you want to make. Run `rails test` to see the tests fail.
+1. Fix stuff.
+1. Run `rails test` to see if the tests pass. Repeat steps 2-4 until done.
+1. Update the documentation and Changelog to reflect any changes.
```
Comment (mikz) [inline CONTRIBUTING.md]: There is no documentation or changelog :)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Contributing
+
+### Code style
+Regarding code style like indentation and whitespace, **follow the conventions you see used in the source already.**
+You can look at `.rubocop.yml` and `.codeclimate.yml`
+
+## Modifying the code
+First, ensure that you have installed Ruby 2.2+ and Postgresql
```
Comment (mikz) [inline CONTRIBUTING.md]: Ruby > 2.3. We probably should lock that in a gemfile.

File: CONTRIBUTING.md (hunk 1 above)
Comment (andrewdavidmackenzie) [reply CONTRIBUTING.md]: that's the first update! :-)

File: CONTRIBUTING.md (hunk 2 above)
Comment (andrewdavidmackenzie) [reply CONTRIBUTING.md]: Postgres min version? 9.5?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+1. Run `bundle install` to install all dependencies.
+
+Assuming that you don't see any red, you're ready to go.
+
+## Submitting pull requests
+
+1. Create a new branch, please don't work in your `master` branch directly.
+1. Add failing tests for the change you want to make. Run `rails test` to see the tests fail.
+1. Fix stuff.
+1. Run `rails test` to see if the tests pass. Repeat steps 2-4 until done.
```
Comment (andrewdavidmackenzie) [inline CONTRIBUTING.md]: steps 2-3 :-)    or it's infinite a loop within a loop :-)

File: CONTRIBUTING.md (hunk 1 above)
Comment (andrewdavidmackenzie) [inline CONTRIBUTING.md]: Maybe something about issues? e.g. look for easy issues to fix? or submit issues for problems you have using the supplied template...?

Comment (mikz) [reply CONTRIBUTING.md]: I think it is fine like this. If there are easy to fix issues we will label them as such and it should be self-describing.

File: CONTRIBUTING.md (hunk 2 above)
Comment (mikz) [reply CONTRIBUTING.md]: Not sure about the Postgres version. So better keep it like this.

**Pull request merged**